### PR TITLE
Hyperledger Labs proposal for Private Data Objects.

### PR DESCRIPTION
**Pull request by cmickeyb**
Signed-off-by: Mic Bowman <mic.bowman@intel.com>

### CODE REVIEW

File: labs/PrivateDataObjects.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Private Data Objects
+
+# Short Description
+
+The Private Data Objects lab provides technology for
+confidentiality-preserving, off-chain smart contracts.
+
+A private data object (PDO) is a stateful "smart contract" that
+implements a set of access and update policies on confidentail data
```
Comment (tkuhrt) [inline labs/PrivateDataObjects.md]: confidential instead of confidentail

**Comment by cmickeyb** [comment]
Spelling error fixed.

**Comment by tkuhrt** [comment]
Based on a conversation with Mic, we would create a new repo for this and the committers would migrate what they want over to it from the existing repo.

LGTM +1

**Comment by trbs** [comment]
LGMT as well +1

**Comment by lehors** [comment]
LGTM +1
Nit: it would probably be good practice to require the sponsor to be identified in an unambiguous way, using their github id, LF id, or email address, even though in this case everybody of course knows who the one and only world-famous Dan Middleton is! ;-)

**Comment by cmickeyb** [comment]
Added Dan's email (although... he really is famous enough!).
Changed the name to something more appropriate for repo naming.

**Pull request merged**